web/src/css/6-components/_know-your-rights-route.scss

```diff
@@ -0,0 +1,21 @@
+.RightsList {
+    font-family: Times;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    padding: 0.6rem;
+}
+
+.RightsItem h6 {
+    font-weight: bold;
+    font-size: 1.2em;
+    padding: 0.6rem;
+    display: block;
+
+}
+
+.RightsItem li {
```

Comment:
Same thing here about the tag selector. Preference to make it a class-based selector.